en/config_vtol/vtol_duo_tailsitter_config.md

```diff
@@ -0,0 +1,98 @@
+# Duo-Tailsitter VTOL Configuration
+
+This is the configuration tuning documentation for a "Duo-Tailsitter VTOL".
+
+A tailsitter is essentially a fixed wing vehicle that takes off in a pitched up configuration (essentially sitting on its tail, hence the name) and then transitions to fixed wing flight by pitch down by 90 degrees to achieve the fixed wing configuration.
+The Duo-Tailsitter variant is so-named because it has two motors (a "quad" tailsitter would have four).
+
+Like other VTOLs, the Duo-Tailsitter has two modes of operation: multicopter (MC) mode and fixed wing (FW) mode.
+In MC mode, lift and roll are dependent on the motors, while the pitch and yaw depend on the elevons.
+In FW mode, yaw depends on differential thrust from the motors, while roll and pitch depend on the elevons.
+
+For airframe specific documentation and build instructions see [VTOL Framebuilds](../frames_vtol/README.md).
+
+## Firmware & Basic Settings
+
+1. Run *QGroundControl*
+2. Flash the firmware for your current release or master (PX4 `main` branch build).
+3. In the [Frame setup](../config/airframe.md) section select the appropriate VTOL airframe.
+   At time of writing, this must be the [Generic VTOL Tailsitter](../airframes/airframe_reference.md#vtol-tailsitter).
+4. Perform all the other [standard configuration](../config/README.md), including compass, radio setup, etc., except for autotuning.
+
+   :::note
+   Autotuning is not supported for a duo tailsitter!
+   :::
+
+### Flight / Transition Mode Switch
+
+You should assign a switch on your RC controller for changing between the multicopter and fixed wing modes.
+
+:::note
+While PX4 allows flight without an RC controller, you must have one when tuning/configuring a new airframe.
+:::
+
+This is done in [Flight Mode](../config/flight_mode.md) configuration, where you [assign flight modes and other functions](../config/flight_mode.md#what-flight-modes-and-switches-should-i-set) to switches on your RC controller.
+The switch can also be assigned using the parameter [RC_MAP_TRANS_SW](../advanced_config/parameter_reference.md#RC_MAP_TRANS_SW).
+
+The switch in the off-position means that you are flying in multicopter mode.
+
+## Duo Tailsitter MC Mode Tuning Guide
+
+MC-mode tuning is dificult because some axes depend on control surfaces, while others depend on the motors.
+
```

Comment:
It would be better to find tests that require no touching of the vehicle. Then we could at least also add - secure vehicle immovably or similar.
@sfuhrer is this warning reasonable/sufficient?

```suggestion

:::warning
Parts of this procedure are very _dangerous_ - they rely on active spinning propellers.

- Remove propellers when not explicitly required.
- Do not perform steps with spinning props unless you can find a way to do them safely.
  Safety glasses, helmet and thick gloves are recommended but may not be sufficient!
:::

```